engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -1851,7 +1961,14 @@ namespace dmGameSystem
 
             float radius_sq = radiuses[entry->m_UserData];
 
-            bool intersect = dmIntersection::TestFrustumSphereSq(frustum, entry->m_WorldPosition, radius_sq);
+            SpriteComponent* component = &sprite_world->m_Components.Get(entry->m_UserData);
```

Comment:
Use pivot point only for frustrum culling because for culling we need the center of the sprite, but for render sorting - just position of the sprite. Fix for https://github.com/defold/defold/issues/11313